src/shared/containers/tc-communities/CommunityStats.jsx

```diff
@@ -13,11 +13,13 @@ import shortid from 'shortid';
 
```

Comment:
The stuff you did in this file, looks correct, but does not work in practices, because not every community, and not every page inside communities, renders the CommunityStats container; thus in most of cases the list of communities visible to the user will never be loaded. You should move this logic into src/shared/containers/tc-communities/Header.jsx (yeah, there you'll have to add the class component wrapper to have `componentDidMount()` hook from where you can fire the get list action, when necessary).